src/engine/strat_engine/backstore/backstore.rs

```diff
@@ -458,6 +458,10 @@ impl Backstore {
         size - self.next
     }
 
+    pub fn available(&self) -> Sectors {
```

Comment:
This needs documenting very carefully. I can't tell what it would eventually mean once other layers are interposed.

Reply:
see d6cbc12, yes this will not work in 2.0.